digitalocean/database/resource_database_user_test.go

```diff
@@ -498,6 +557,60 @@ resource "digitalocean_database_user" "foobar_user" {
   }
 }`
 
+const testAccCheckDigitalOceanDatabaseUserConfigOpenSearchACL = `
+resource "digitalocean_database_cluster" "foobar" {
+  name       = "%s"
+  engine     = "opensearch"
+  version    = "2"
+  size       = "db-s-2vcpu-2gb"
```

Comment:
This doesn't seem to be a valid size for an OpenSearch cluster:

```
        Error: Error creating database cluster: POST https://api.digitalocean.com/v2/databases: 422 (request "822ac77c-4ce2-489f-9d3d-838e22475d6a") invalid size
        
          with digitalocean_database_cluster.foobar,
          on terraform_plugin_test.tf line 2, in resource "digitalocean_database_cluster" "foobar":
           2: resource "digitalocean_database_cluster" "foobar" {
```

Can you please ensure your new acceptance test passes. You can run just your test with:

```
make testacc TESTARGS='-run=TestAccDigitalOceanDatabaseUser_OpenSearchACLs'
```